scripts/install_third_party_libs.py

```diff
@@ -265,18 +283,24 @@ def main(args=None):
     python_utils.PRINT('Installing third-party JS libraries and zip files.')
     install_third_party.main(args=[])
 
+    if common.is_windows_os():
+        tweak_yarn_executable()
+
     # Install third-party node modules needed for the build process.
-    subprocess.check_call(['yarn'])
+    subprocess.check_call([get_yarn_command()])
 
     install_skulpt(parsed_args)
 
     # Install pre-commit script.
     python_utils.PRINT('Installing pre-commit hook for git')
     pre_commit_hook.main(args=['--install'])
 
-    # Install pre-push script.
-    python_utils.PRINT('Installing pre-push hook for git')
-    pre_push_hook.main(args=['--install'])
+    # TODO(cuichenli): Once pre_commit_linter is working correctly, this
```

Comment:
Please link to an issue tracking this and assign it to yourself.

Our TODO syntax is now "TODO(#issueNum)".

Reply:
Updated